### ISSUE TITLE
Keyboard shortcut to temporarily disable filtering on current page for Firefox?

### ISSUE THREAD

**Issue by vibrasphere** [gorhill/uMatrix#453]
E.g. NoScript use CTRL+Shift+\ to disable it on current page.

Really tired hovering over uMatrix icon clicking Disable/Enable Matrix Filtering and then clicking Reload page for every page. And I'm not talking about websites that you visit daily, you just save rules for it, but I mean websites that you visit quickly for an article, etc. and never visit it back. One time websites, e.g. Googling something and ending up on some website that you don't use, except for that one time and each time you need to play with uMatrix manually.

Keyboard shortcut would save a lot of time!

**Comment by gorhill** [comment]
Just curious: do you try to at least toggling the `all` cell to `allow` before disabling matrix filtering?

**Comment by vibrasphere** [comment]
Why? So that's like 3 clicks now. Even more work.

**Comment by gorhill** [comment]
I am not arguing with providing a shortcut, I am just asking because disabling matrix filtering for an unknown site is a rather drastic choice compared with toggling the `all` cell -- which will keep you protected from all the blacklisted domains.

**Comment by AdKiller** [comment]
yes please, keyboard shortcuts for every common task. I'm dying here with clicking the mouse...

**Comment by vibrasphere** [comment]
So what's the status for a shortcut to temp disable 'all' + 'iframe' with a keybord shortcut?
